compiler/bindings/python/test/ir/dialects_test.py

```diff
@@ -232,3 +232,44 @@ def compilation_info():
     assert compilation_info is not None
     assert compilation_info.lowering_config == lowering_config
     assert compilation_info.translation_info == translation_info
+
+
+@run
+def test_query_mma():
+    test_module = ir.Module.parse(
+        """
```

Comment:
I'm concerned this is a 'change detector' test. Every time we update any of the related dialects, we will have to come back to this test and also update it. It's worse than lit tests were at least you have the lsp helping you with syntax highlighting etc. IMO we can do without a test here -- the logic is tested on the C++ side with your test pass.